### ISSUE TITLE
Importing JSON files?

### ISSUE THREAD

**Issue by beck** [whatwg/loader#133]
Led here from:
https://esdiscuss.org/topic/importing-json-files

Looking through the issues and discussion, unable to find a clear answer.

There is a ticket discussing [Import non-JS code from the import declaration](https://github.com/whatwg/loader/issues/87).

Considering ubiquity of JSON, has there been any discussion on giving it native support?

**Comment by guybedford** [comment]
This sort of thing can be done through a custom format loading with the loader hooks.

In SystemJS the above method is provided via `SystemJS.config({ meta: { 'app/*.json': { format: 'json' } } })`.

Any spec inclusion is of course separate to the above, but just to mention this approach.

**Comment by caridy** [comment]
yeah. I suspect that the default loader implemented in each engine will support `json`, but we haven't get to define that just yet.
